### PR TITLE
localFetcher

### PR DESCRIPTION
**Pull request by frodare**
Adding localFetcher, a function that adapts local data stored in an array to the fetchRows and fetchCount methods required for useManifest

### CODE REVIEW

File: src/utils/localFetcher.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  if (typeof nameB === 'string') nameB.toUpperCase()
+  if (nameA < nameB) return -1
+  if (nameA > nameB) return 1
+  return 0
+}
+
+const KEEP_ALL = () => true
+
+export default (data, filterer = KEEP_ALL) => {
+  const preprocess = (filter, tableViewState) => data.filter(filterer).sort(sorter(tableViewState))
```
Comment (zeriley) [inline src/utils/localFetcher.js:38]: What happens when .sort takes in undefined?

Comment (zeriley) [reply src/utils/localFetcher.js:38]: Just acts like .sort() would?

**Review by zeriley — APPROVED** [review]

**Review by mrprim — APPROVED** [review]

File: src/utils/localFetcher.js (hunk 1 above)
Comment (frodare) [reply src/utils/localFetcher.js:38]: we are always passing it a function here

Comment (zeriley) [reply src/utils/localFetcher.js:38]: But the function returns undefined if no sort exists

Comment (frodare) [reply src/utils/localFetcher.js:38]: that would have been helpful info in the first comment, lol, not sure what would happen there.

**Pull request merged**